private/projects/pdk-monorepo-project.ts

```diff
@@ -90,33 +54,26 @@ export class PDKMonorepoProject extends NxMonorepoProject {
         syncpackConfig: { ...DEFAULT_CONFIG, workspace: false },
       },
       deps: ["fast-xml-parser", "projen", "@pnpm/types@^9.0.0"],
-      nxConfig: {
-        // This is OK to be stored given its read only and the repository is public
-        nxCloudReadOnlyAccessToken:
-          "OWJmZDJmZmEtNzk5MC00OGJkLTg3YjUtNmNkZDk1MmYxZDZkfHJlYWQ=",
-        cacheableOperations: ["build", "test", "generate"],
-        targetDefaults: NX_TARGET_DEFAULTS,
-        targetDependencies: {
-          "release:mainline": [
-            {
-              target: "release:mainline",
-              projects: Nx.TargetDependencyProject.DEPENDENCIES,
-            },
-          ],
-          upgrade: [
-            {
-              target: "upgrade",
-              projects: Nx.TargetDependencyProject.DEPENDENCIES,
-            },
-          ],
-        },
-      },
       workspaceConfig: {
         disableNoHoistBundled: true,
         linkLocalWorkspaceBins: true,
       },
     });
 
+    // Turn on automatic target inference for this repo
+    this.nx._autoInferProjectTargets = true;
+    // This is OK to be stored given its read only and the repository is public
+    this.nx.useNxCloud(
+      "OWJmZDJmZmEtNzk5MC00OGJkLTg3YjUtNmNkZDk1MmYxZDZkfHJlYWQ="
+    );
+    this.nx.cacheableOperations.push("generate");
```

Comment:
Can we also set the cacheDirectory to `.nx/cache`?

Reply:
Ah yes, forgot about that